skimage/measure/fit.py

```diff
@@ -504,24 +721,33 @@ class EllipseModel(BaseModel):
     solution is computed directly, no iterations are required. This leads
     to a simple, stable and robust fitting method.
 
-    The ``params`` attribute contains the parameters in the following order::
-
-        xc, yc, a, b, theta
-
-    Attributes
+    Parameters
     ----------
-    params : tuple
-        Ellipse model parameters in the following order `xc`, `yc`, `a`, `b`,
-        `theta`.
+    center : array-like, shape (2,)
+        Coordinates of ellipse center.
+    ax_lens : array-like, shape (2,)
```

Comment:
Is this a typo? Shouldn't it be `ax_len` or `ax_length`? With "lens" I think of an "optical lens" first.

Reply:
It’s short for “axis lengths” (there are two axes).   Do you prefer the longer form?  Or something else?